src/nvim/terminal.c

```diff
@@ -154,6 +151,8 @@ static Map(int, int) *color_indexes;
 static int default_vt_fg, default_vt_bg;
 static VTermColor default_vt_bg_rgb;
 
+// init/teardown {{{
```

Comment:
please don't add these kinds of folding things to the source. They are just more stuff that needs to be maintained.